packages/cli/src/commands/spaces/outbound-rules/index.ts

```diff
@@ -0,0 +1,51 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import heredoc from 'tsheredoc'
+import {displayRules, displayRulesAsJSON} from '../../../lib/spaces/outbound-rules'
+import * as Heroku from '@heroku-cli/schema'
+import {SpaceCompletion} from '@heroku-cli/command/lib/completions'
+
+export default class Index extends Command {
+  static description = heredoc(`
+    list Outbound Rules for a space
+    Outbound Rules are only available on Private Spaces.
+
+    Newly created spaces will have an "Allow All" rule set by default
+    allowing all egress dyno traffic outside of the space.  You can
+    remove this default rule to completely stop your private dynos from
+    talking to the world.
+  `)
```

Comment:
Description is missing a line of text:

```suggestion

    You can add specific rules that only allow your dyno to communicate with trusted hosts.
  `)
```